UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Validation/V2/UpdatesSinceValidator.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.Extensions.Configuration;
+using UKHO.ExchangeSetService.Common.Extensions;
+using UKHO.ExchangeSetService.Common.Models.Enums;
+using UKHO.ExchangeSetService.Common.Models.V2.Request;
+
+namespace UKHO.ExchangeSetService.API.Validation.V2
+{
+    public class UpdatesSinceValidator : AbstractValidator<UpdatesSinceRequest>, IUpdatesSinceValidator
+    {
+        private readonly IConfiguration _configuration;
+        private DateTime _sinceDateTime;
+
+        public UpdatesSinceValidator(IConfiguration configuration)
+        {
+            _configuration = configuration;
+
+            RuleFor(x => x.SinceDateTime)
+                .Must(x => x.IsValidRfc1123Format(out _sinceDateTime))
```

Comment:
Valid date is in "yyyy-MM-ddTHH:mm:ss.fffZ" format (ISO 8601) please change wherever applicable.

Reply:
Resolved